jcl/src/java.base/share/classes/com/ibm/sharedclasses/spi/SharedClassProvider.java

```diff
@@ -228,4 +228,9 @@
 	 * 					A new instance of SharedClassPermission which is a sub-class of BasicPermission, or null if shared classes are not enabled.
 	 */
 	public BasicPermission createPermission(String classLoaderClassName, String actions);
+	
+	/**
+	 * <p>notify the close state to all classpaths related to this URLClassLoader.</p>
```

Comment:
Better to change 

"all classpaths related to this URLClassLoader" to "all the jar/zip files on the URL class path of the class loader that uses SharedClassProvider".

Reply:
Already updated the description as suggestions.